content/en/_redirects

```diff
@@ -91,7 +91,7 @@ docs/started/requirements/                    /docs/started/getting-started/
 # Restructured the getting-started and other-guides sections.
 /docs/started/getting-started-k8s/             /docs/started/k8s/
 /docs/started/getting-started-minikf/                      /docs/started/workstation/getting-started-minikf/
-/docs/started/getting-started-minikube/                    /docs/started/workstation/minikube-linux/
+/docs/started/getting-started-minikube/                    /docs/started/methods/kfctl/minikube/
```

Comment:
I believe for each file move, you need to set up a new redirection rule, because there can be links around the internet pointing to these docs.

Reply:
Hey @Bobgy! That's a very good point! Just created #2511 to tackle this

Reply:
This issue should not be addressed as a separate issue. Each move must contain the relevant redirects to keep from breaking incoming links and on-site links.

Reply:
That is a good point! Let's merge first the redirects change and then this one.

I will keep them separate so it's easier to review, this one is starting to be big. :)

Reply:
[Forget prior msg] 

I think it's easier to do as you propose. Include redirects on each change, so it's done in bite-sized chunks

Reply:
Added redirects in these commits - [operator](https://github.com/kubeflow/website/pull/2492/commits/54e2b002e4b340fe7a34e7790c43ef68d6c3af47), [kfctl](https://github.com/kubeflow/website/pull/2492/commits/fedabc8f93e1a5f471ac472c81b1b4390e16eb1d)